crates/worker/src/partition/state_machine/mod.rs

```diff
@@ -891,7 +896,23 @@ impl<Codec: RawEntryCodec> StateMachine<Codec> {
                 )
                 .await?
             }
-            _ => {
```

Comment:
This is a really good catch. Personally, I would like to forbid wildcard patterns like this because it just leads to forgetting to updating some parts of the code base.

Reply:
Yeah I think we should really never use the wildcard here in the state machine, it's very easy to mess this up